src/security/ServerOptions.cc

```diff
@@ -280,7 +286,10 @@ Security::ServerOptions::createSigningContexts(const AnyP::PortCfg &port)
     // contexts are generated as needed. This method initializes the cert
     // and key pointers used to sign those contexts later.
 
-    signingCa = certs.front();
+    // backward compatibility for old configs using cert= as signing CA.
```

Comment:
Add a deprecation warning instead of a comment.

Reply:
that is done in the ::parse() method now.